examples/tensorflow/sentiment-analyzer/predictor.py

```diff
@@ -0,0 +1,29 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
+
+import tensorflow as tf
+import tensorflow_hub as hub
+from bert import tokenization, run_classifier
+
+labels = ["negative", "positive"]
```

Comment:
Should this be in `init()`? I'm not sure that it should, just wanted you to consider